src/app/inbox/components/notification-your-record-amended/notification-your-record-amended.component.html

```diff
@@ -5,10 +5,10 @@
 
 <!--  https://github.com/ORCID/ORCID-Source/blob/a9868bbc959b151d7c761936b610936faf1b2d84/orcid-web/src/main/resources/freemarker/includes/ng2_templates/notification-body-ng2-template.ftl#L81 -->
 
-<h3>
+<div>
   has updated the {{ getNotificationSectionUpdatedLabel(notification) }} section
```

Comment:
@leomendoza123 all good with the PR! I'm just wondering why this section is not internationalized?